src/components/uicomposition/LayoutSection.astro

```diff
@@ -0,0 +1,20 @@
+---
+interface Props {
+    backgroundColor?: "light" | "dark";
+    class?: string;
+}
+
+const { 
+    backgroundColor = "light",
+    class: className,
+} = Astro.props;
+---
+<section class:list={[
```

Comment:
Can we use a non semantic element here? because I fear it will not be used in a semantic way.
We are only using the element as a styling wrapper, so it is safer to use use a [<div>]